### DISCUSSION TITLE
Read the config from yml file without being logged into the backend

### DISCUSSION THREAD

**Discussion by yadsamit-disc** [pulumi/pulumi#7413] [Pulumi in Practice] [answered]
Hello,

**Current understanding:**
In order to be able to fetch config from the yaml file using the command : `pulumi config get <key>`
- we need to be logged into the backend
- the state of the stack for which we need the config has to be initialized already - `pulumi stack init <stackname>`

**Question:**
Is it possible to fetch the config from yml file ?
1. without being logged in to the backend 
2. If login is required then, without needing to initiate the stack

Could this be a feature if above is not true ?

**Comment by rawkode** [comment, marked as answer]
This isn't possible without a stack, because the config exists within the stack file (eg: `Pulumi.stackname.yaml`

You don't need to be logged into the backend, this works with `pulumi login --local` also.
